styles/style.css

```diff
@@ -0,0 +1,71 @@
+
+
+body {
+    font-family: 'Krona One';font-size: 22px;
+    background-color:#EFEFD0
+}
+h1 {
+ font-style: strong
+
+}
+
+#container {
+  border: 5px solid #ffcc5c;
+  display: flex;
+  flex-direction: column ;
+  justify-content: space-around;
+align-items: center;
+
+}
+
+ul{
+list-style-type: none;
+}
+
+nav ul {
+  display: flex;
+  flex-direction: row  ;
+  justify-content: space-around;
```

Comment:
I liked how you used Flexbox for this.  I'm wondering, though, if flexbox will allow you to push two of the items to the left and then one of them to the right?!?!